<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Парнас 0.9</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="PropWidget" name="wgtProp" native="true">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Исходные данные</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Название анализируемой величины:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txtPropertyName"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="SourceDataView" name="tblSourceData">
                <property name="selectionBehavior">
                 <enum>QAbstractItemView::SelectRows</enum>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="hlayStat">
                <item>
                 <layout class="QHBoxLayout" name="hlayMean">
                  <item>
                   <widget class="QLabel" name="label_3">
                    <property name="text">
                     <string>Среднее значение:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="lblMean">
                    <property name="text">
                     <string>0</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_2">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="hlayStdev">
                  <item>
                   <widget class="QLabel" name="label_5">
                    <property name="text">
                     <string>СКО:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="lblStdev">
                    <property name="text">
                     <string>0</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_3">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="hlayMin">
                  <item>
                   <widget class="QLabel" name="lblLabelMin">
                    <property name="text">
                     <string>Min:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="lblMin">
                    <property name="text">
                     <string>0</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_4">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="hlayMax">
                  <item>
                   <widget class="QLabel" name="lblLabelMax">
                    <property name="text">
                     <string>Max:</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="lblMax">
                    <property name="text">
                     <string>0</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer_5">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <property name="spacing">
               <number>3</number>
              </property>
              <item>
               <widget class="QPushButton" name="btnAddValue">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>50</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="text">
                 <string>Добавить значение</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnDeleteValue">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="text">
                 <string>Удалить</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnExceptValue">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="text">
                 <string>Исключить</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="gbxBorders">
                <property name="maximumSize">
                 <size>
                  <width>160</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="title">
                 <string>Границы диапазона</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout">
                 <item>
                  <widget class="QCheckBox" name="cbxLower">
                   <property name="text">
                    <string>Нижняя</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="DoubleNumberEdit" name="txtLower">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>150</width>
                     <height>16777215</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QCheckBox" name="cbxUpper">
                   <property name="text">
                    <string>Верхняя</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="DoubleNumberEdit" name="txtUpper">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>150</width>
                     <height>16777215</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="btnApply">
                   <property name="maximumSize">
                    <size>
                     <width>150</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Применить</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Результаты</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="ResultsWidget" name="results" native="true"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="btnSavePropertyChanges">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Сохранить изменения</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnRollbackPropertyChanges">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Отменить изменения</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DoubleNumberEdit</class>
   <extends>QLineEdit</extends>
   <header>extra_widgets/numberedit.h</header>
  </customwidget>
  <customwidget>
   <class>ResultsWidget</class>
   <extends>QWidget</extends>
   <header>resultswidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SourceDataView</class>
   <extends>QTableView</extends>
   <header>sourcedata.h</header>
  </customwidget>
  <customwidget>
   <class>PropWidget</class>
   <extends>QWidget</extends>
   <header>propwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>cbxUpper</sender>
   <signal>toggled(bool)</signal>
   <receiver>txtUpper</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>818</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>838</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbxLower</sender>
   <signal>toggled(bool)</signal>
   <receiver>txtLower</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>808</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>821</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>sgnlChanged()</signal>
  <slot>slotAddProperty()</slot>
  <slot>slotDeleteProperty()</slot>
  <slot>slotSaveProperty()</slot>
  <slot>slotSourceDataDelete()</slot>
  <slot>slotDataChanged()</slot>
 </slots>
</ui>
